ontology_subclass_of_topic: /ric/knowrob/subclasses_of
ontology_superclass_of_topic: /ric/knowrob/superclasses_of 
ontology_create_instance_topic: /ric/knowrob/createInstance
ontology_dump_ontology_topic: /ric/knowrob/dumpOntology
ontology_load_ontology_topic: /ric/knowrob/loadOntology
ontology_user_instances_from_class_topic: /ric/knowrob/userInstancesFromClass
